manifests/0000_70_cluster-network-operator_01_pki_crd.yaml

```diff
@@ -20,21 +20,38 @@ spec:
   - name: v1
     schema:
       openAPIV3Schema:
-        description: "OperatorPKI is a simple certificate authority. It is not intended for external use - rather, it is internal to the network operator. The CNO creates a CA and a certificate signed by that CA. The certificate has both ClientAuth and ServerAuth extended usages enabled. \n  More specifically, given an OperatorPKI with <name>, the CNO will manage: - A Secret called <name>-ca with two data keys:   - tls.key - the private key   - tls.crt - the CA certificate - A ConfigMap called <name>-ca with a single data key:   - cabundle.crt - the CA certificate(s) - A Secret called <name>-cert with two data keys:   - tls.key - the private key   - tls.crt - the certificate, signed by the CA \n The CA certificate will have a validity of 10 years, rotated after 9. The target certificate will have a validity of 6 months, rotated after 3 \n The CA certificate will have a CommonName of \"<namespace>_<name>-ca@<timestamp>\", where <timestamp> is the last rotation time."
+        description: "OperatorPKI is a simple certificate authority. It is not intended
+          for external use - rather, it is internal to the network operator. The CNO
+          creates a CA and a certificate signed by that CA. The certificate has both
+          ClientAuth and ServerAuth extended usages enabled. \n  More specifically,
+          given an OperatorPKI with <name>, the CNO will manage: - A Secret called
+          <name>-ca with two data keys:   - tls.key - the private key   - tls.crt
+          - the CA certificate - A ConfigMap called <name>-ca with a single data key:
+          \  - cabundle.crt - the CA certificate(s) - A Secret called <name>-cert
+          with two data keys:   - tls.key - the private key   - tls.crt - the certificate,
+          signed by the CA \n The CA certificate will have a validity of 10 years,
+          rotated after 9. The target certificate will have a validity of 6 months,
+          rotated after 3 \n The CA certificate will have a CommonName of \"<namespace>_<name>-ca@<timestamp>\",
+          where <timestamp> is the last rotation time."
         properties:
           apiVersion:
-            description: 'APIVersion defines the versioned schema of this representation of an object. Servers should convert recognized schemas to the latest internal value, and may reject unrecognized values. More info: https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#resources'
+            description: 'APIVersion defines the versioned schema of this representation
+              of an object. Servers should convert recognized schemas to the latest
+              internal value, and may reject unrecognized values. More info: https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#resources'
```

Comment:
Not that I really care, but since we're splitting into shorter lines, probably the More info link makes the line too long and should be yet another line.